src/pages/Architecture/index.js

```diff
@@ -47,11 +49,15 @@ function ArchitecturesSection() {
     let location = useLocation();
     const history = useHistory();
 
-    const { parentId } = useParams();
-
+    var { parentId } = useParams();
     const { isOpen, onOpen, onClose } = useDisclosure();
+    const { initialized, keycloak } = useKeycloak();
     const cancelRef = React.useRef();
 
+    if (keycloak?.tokenParsed?.given_name === 'Admin' && location.pathname === '/architectures') {
```

Comment:
We can go with the give_name then only the person with the name admin can create the projects